docs/readme.md

```diff
@@ -0,0 +1,9 @@
+# Taxonomy
+
+## Unclassified Tree
+
+Nodes are deemed *unclassified* when their *scientific name* contains (ignore case) the word `unclassified`; these generate the **Unclassified** tree.
```

Comment:
@agarciamontoro I'm assuming the meaning of *generate* here is common knowledge; add it here if you want.

Reply:
It's ok as it is now. Previous comment addresses this.